Rakefile

```diff
@@ -56,6 +56,18 @@ namespace :test do
     t.verbose    = true
   end
 
+  Rake::TestTask.new('plugins:avatax') do |t|
+    t.libs << BASE_DIR
+    t.test_files = FileList["#{BASE_DIR}/plugins/avatax/test_*.rb"]
+    t.verbose    = true
+  end
+
+  Rake::TestTask.new('plugins:killbill-email-notifications') do |t|
```

Comment:
I think `test:all` will need to be updated as well to run these.

Also, can you try running that full suite against a fresh Kill Bill installation (no plugin), to make sure no other test makes assumptions regarding the already available plugins?